rero_ils/modules/mef_persons/templates/rero_ils/detailed_view_persons.html

```diff
@@ -51,6 +51,22 @@ <h1 class="mb-0">{{ record | person_label(current_i18n.language) }}</h1>
     </section>
   </article>
 </article>
+<!-- Documents -->
+{%- if record.documents %}
+<section class="py-4 border-top">
```

Comment:
py-4 is too much, pt-4 is enough, I think.

Reply:
Changed